src/translators/sql_translator.py

```diff
@@ -1009,6 +1009,18 @@ def delete_entity(self, entity_id, entity_type=None, from_date=None,
 
         # First delete entries from table
         table_name = self._et2tn(entity_type, fiware_service)
+
+        # how many rows in total?
+        try:
+            self.cursor.execute("select count(*) from {}".format(table_name))
+        except Exception as e:
+            logging.error("{}".format(e))
+            return 0
+        # TODO why the result still keeps into account the deleted rows???
+        # this query was moved up to make it "consistent" also with db that
```

Comment:
I tried to reproduce this but it never happened for me, i.e. the count from the select = total rows in the table before deleting and `deleted_rows` = rows actually deleted. Surely this is a possibility in general though if this API calls happens concurrently with a notify call containing entities to be inserted in the same table!